docs/finalproject/external-projects.md

```diff
@@ -10,9 +10,14 @@ If you are one of these charities, or know one we could work with, you can find
 
 ## Past Projects
 
-Here is an example of a web app we built with an NGO as a Final Project:
+Here are examples of web apps we built as a Final Project:
 
 - [Over The Line with The Compass Collective](https://over-the-line.uk/)
+- [Chatterbox Talents](https://ldn8-horizon-chatterbox-talent.netlify.app/)
+- [Hujreh](https://hujreh.netlify.app/)
+- [The Breteau Foundation](https://bugsquashers-edu.netlify.app/)
+- [Pratibha Nakshtra](https://pratibha-nakshtra-teamhapi.netlify.app/)
```

Comment:
```suggestion

```
Also really unfinished with dummy text. I think we have better examples than this. Can we use the Bikes For Refugees project from GL5, Energiser from WM3,  and Goal from NW4? (Not blocking, as an enhancement

https://linktr.ee/callbackcats - GL5 contacts to get that back up
https://github.com/musayuksel/Goal-app-CYF-final-project 
I wish I could find the WM3 project but I have trawled a few thousand files and I cannot find the WM3 Final Projects tracker anywhere. It's pretty impossible for me to find anything on drive without knowing the exact name or link already as I own all the trainee documents.

They would need to be redeployed on render.com but those are good looking, finished applications and would be a much better example.